### ISSUE TITLE
Missing info off artwork page

### ISSUE THREAD

**Issue by ZiggyAtl** [ocupop/comission-application#152]
@paulstroot sorry mate, just found another 2 bugs which have crept in (probably related). On recent artwork that's been added it is doing two things on the individual artwork page which aren't correct;

- [ ] It's stopped showing the information underneath the image allowing the user to view dabs, price tiers, unavailable etc. Screenshot below is of one that is working correctly to illustrate the info I'm referring to.
![Screen Shot 2022-11-10 at 8 45 35 AM](https://user-images.githubusercontent.com/112654923/201109865-81b13f8c-35f7-4838-a84b-df3f1c427170.png)

- [ ] When you move the cursor over the image, a grey instruction box about clicking and dragging to add tiles comes up.
![Screen Shot 2022-11-10 at 8 41 46 AM](https://user-images.githubusercontent.com/112654923/201110094-3ad1c0fa-2997-494b-9321-8e80c6c2923a.png)

**Comment by paulstroot** [comment]
Question about the first item:

> `stopped showing the information underneath the image allowing the user to view dabs, price tiers, unavailable etc. Screenshot below is of one that is working correctly to illustrate the info I'm referring to.`

I don't have any clear direction on what to place on this artwork details page, and I'm not sure what is within scope of this first round of our project. Looping @tbeck in on this question as well.

**Comment by ZiggyAtl** [comment]
@paulstroot sorry for the confusion. What I was saying was that the box which was previously under the artwork in the app (see first screen shot), allowing the museum to show dab borders, show pricing tiers and show unavailable dabs (which will be very useful for them) has disappeared on some of the artworks (see second screen shot)

**Comment by ZiggyAtl** [comment]
@paulstroot I noticed that the info box allowing the museum to show dab borders and show pricing tiers has returned (see original screenshot above), but not the one to show unavailable DABs

**Comment by paulstroot** [comment]
The "Show DAB Borders" and "Show Pricing tiers" toggles are now on the art details page.

We have decided not to show unavailable tiles to museum admins because that would create more confusion for them.

FYI, I cleaned out our test products, so feel free to add some more as you continue to test.

![image](https://user-images.githubusercontent.com/52838369/201707700-4b661537-84ba-4164-8659-5e63a78ea10b.png)
